wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -57,6 +57,12 @@ public CaptureWindow()
                 using (var ctx = new Context())
                 {
                     var devices = ctx.QueryDevices();
+                    if (0==devices.Count)
+                    {
+                        Console.WriteLine("The tutorial {0} requires Realsense D400 device to run.\nConnect a device and rerun",
+                            System.Diagnostics.Process.GetCurrentProcess().ProcessName);
+                        Environment.Exit(0);

```

Comment:
Shouldn't it be 1?

Reply:
Missing a proper camera is not an application error, but I'll put 1 anyway

Reply:
I don't see it...
Anyway, any abnormal termination (including invalid usage) should exit with non-0